pages/acknowledgments.tex

```diff
@@ -36,37 +38,42 @@ \section*{Acknowledgments}
 
 ~\\\noindent People that have been working on this rulebook as member of one of the league's committees (in alphabetical order):
 \begin{multicols}{3}%
+% MAURICIO MATAMOROS @2018: Leaving only Trustees and Execs as TC will be renewed
 \noindent%
 Loy van Beek\\
 Kai Chen\\
-Kathrin Evers\\\columnbreak
+Justin Hart\\
+\columnbreak
 Dirk Holz\\
 Luca Iocchi\\
 Mauricio Matamoros\\
-Alexander Moriarty\\\columnbreak
+Raphael Memmesheimer\\
+Hideaki Nagano\\
+\columnbreak
+Sammy Pfeiffer\\
 Caleb Rascon\\
 Komei Sugiura\\
-Rodrigo Ventura\\
 Sven Wachsmuth\\
 Tijn van der Zant\\
 \end{multicols}
 
 We also like to thank all the people who contributed to the RoboCup@Home league with their feedback and comments. 
 
+% MAURICIO MATAMOROS @2018: Dummy names. Please addd the contributor username after the @
 ~\\\noindent People that have been working on this rulebook as member the league (in alphabetical order):
 \begin{multicols}{3}%
 \noindent%
-Yuki Furuta\\%                     @furushchev
-Noe Hernandez Sanchez\\%           @nohernan
+Erika Mustermann\\%                @ Name place-holder (German)
```

Comment:
K. I'm not sure if I'm following you but I commented out everything. In any case I'll modify it when the final version is about to be released.